packages/ui/src/elements/WhereBuilder/Condition/index.tsx

```diff
@@ -51,6 +52,7 @@ export const Condition: React.FC<Props> = (props) => {
   const reducedField = reducedFields.find((field) => field.value === fieldName)
 
   const [internalValue, setInternalValue] = useState<string>(value)
+  const [currentOperator, setCurrentOperator] = useState<Operator>(operator)
```

Comment:
I don't think we need additional state for this. Do we?

Reply:
Removed